src/oas/schema/unmarshalers.py

```diff
@@ -85,6 +82,17 @@ def _unmarshal_object(self, instance, schema):
                 except KeyError:
                     continue
             result[name] = self._unmarshal(value, sub_schema)
+
+        additional_properties = schema.get('additionalProperties')
```

Comment:
Based on https://github.com/grktsh/python-oas/issues/5#issuecomment-488287591, should this line rather be as follows?

```python
additional_properties = schema.get('additionalProperties', True)
```

Reply:
LGTM